python/ray/data/tests/test_dataset.py

```diff
@@ -1517,26 +1517,28 @@ def test_dataset_repr(ray_start_regular_shared):
     assert repr(ds) == "Dataset(num_blocks=10, num_rows=10, schema=<class 'int'>)"
     ds = ds.map_batches(lambda x: x)
     assert repr(ds) == (
-        "MapBatches\n" "+- Dataset(num_blocks=10, num_rows=10, schema=<class 'int'>)"
+        "Mapbatches(<Lambda>)\n"
```

Comment:
```suggestion
        "MapBatches(<Lambda>)\n"
```

We should use either snake or camel case here I think.